lib/Makefile

```diff
@@ -19,8 +20,12 @@ ldflags-so		+= -lprotobuf-c
 lib/c/$(CRIU_SO): lib/c/built-in.o
 	$(call msg-link, $@)
 	$(Q) $(CC) -shared $(cflags-so) -o $@ $^ $(ldflags-so) $(LDFLAGS)
+lib/c/$(CRIU_A): lib/c/built-in.o
+	$(call msg-link, $@)
+	$(Q) $(AR) rcs $@ $^
 lib-c: lib/c/$(CRIU_SO)
-.PHONY: lib-c
+lib-a: lib/c/$(CRIU_A)
```

Comment:
don't you need to add it to `all-y` to build .a?

Reply:
yes, it should be added into `all-y` target in the same file

Reply:
Probably, it's also worth to add `install` rule if we plan providing .a library.

Reply:
Thabjs for feedback! I've added a commit.